Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -587,7 +587,10 @@ public void ExecuteActions()
                 expectedStates.Count,
                 _blockChain.Store.GetBlockStates(block1.Hash).Count);
             Assert.Equal(
-                expectedStates.ToImmutableDictionary(),
+                expectedStates.ToImmutableDictionary(
+                    kv => kv.Key.ToHex().ToLowerInvariant(),
```

Comment:
How about using `.ToString()` instead of `.ToHex()`? it adds `0x` as prefix so we can determine easily.

Reply:
That might a good idea, but we eventually remove all these `Address` → `string` conversions in this month.

Reply:
👌